packages/cli/src/commands/proxy/connect.ts

```diff
@@ -0,0 +1,127 @@
+import { ux, Args, Flags } from '@oclif/core'
+import { set } from 'lodash'
+import { tunnelServerFlags, urlFlags } from '@preevy/cli-common'
+import { commands, profileStore, withSpinner, SshConnection } from '@preevy/core'
+import { inspect } from 'util'
+import { formatPublicKey } from '@preevy/common'
+import { spawn } from 'child_process'
+import { connectToTunnelServerSsh } from '../../tunnel-server-client'
+import ProfileCommand from '../../profile-command'
+import { filterUrls, printUrls } from '../urls'
+
+// eslint-disable-next-line no-use-before-define
+export default class Connect extends ProfileCommand<typeof Connect> {
+  static description = 'Tunnel existing local compose application'
+
+  static flags = {
+    ...tunnelServerFlags,
+    ...urlFlags,
+    ...ux.table.flags(),
+    'private-env': Flags.boolean({
+      description: 'Mark all services as private',
+      default: false,
+      required: false,
+    }),
+  }
+
+  static strict = false
+  static hidden = true
+
+  static args = {
+    'compose-project': Args.string({
+      description: 'Compose Project name',
+      required: true,
+    }),
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  async run(): Promise<unknown> {
```

Comment:
If I'm looking at other command, the UI/UX stuff is in the cli/src/commands folder and the business logic is in the core/src folder, consider moving the logic there